grpc/weaviate.proto

```diff
@@ -158,6 +159,19 @@ message HybridSearchParams {
   FusionType fusion_type = 5;
 }
 
+message NearTextSearchParams{
+  message Move{
+    float force =1;
+    repeated string concepts = 2;
+    repeated string uuids = 3;
+  }
+
+  repeated string queries = 1;
+  optional double certainty = 2;
+  optional double distance = 3;
+  Move move_to = 5;
+  Move move_away = 6;
```

Comment:
I would also mark this as `optional`, and please fix the numbers :) should be 4,5 not 5,6